src/components/Card/CardBody.js

```diff
@@ -10,7 +10,7 @@ CardBody.propTypes = {
 
 const Container = styled.div`
 	position: relative;
-	padding: 16px 12px 18px 12px;
+	padding: ${props => props.padding || '16px 12px 18px 12px'};
```

Comment:
I think its just better to do `StyledCardBody(Card.Body)...`
Lets use props for things are that reusable across the app. (for example, `size`, etc)